packages/lastfm/lastfm.0.3.4/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "The lastfm library is an implementation of the API used by the last.fm to keep count of played songs"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Romain Beauxis <toots@rastageeks.org>"]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/savonet/ocaml-lastfm"
+bug-reports: "https://github.com/savonet/ocaml-lastfm/issues"
+depends: [
+  "xmlplaylist"
+  "re" {>= "1.11.0"}
+  "dune" {>= "3.14" & >= "2.0"}
+  "odoc" {with-doc}
```

Comment:
```suggestion
  "ocaml" {>= "4.08"}
  "dune" {>= "3.14"}
  "xmlplaylist"
  "re" {>= "1.11.0"}
  "odoc" {with-doc}
```
For
```
#=== ERROR while compiling lastfm.0.3.4 =======================================#
# context              2.2.0~beta2~dev | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(https://github.com/savonet/ocaml-lastfm/archive/refs/tags/v0.3.4.tar.gz)
# path                 ~/.opam/4.07/.opam-switch/build/lastfm.0.3.4
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p lastfm -j 255 @install
# exit-code            1
# env-file             ~/.opam/log/lastfm-7-602f06.env
# output-file          ~/.opam/log/lastfm-7-602f06.out
### output ###
# (cd _build/default && /home/opam/.opam/4.07/bin/ocamldep.opt -modules -intf src/lastfm_generic.mli) > _build/default/src/.lastfm.objs/lastfm_generic.intf.d
# File "src/lastfm_generic.mli", line 308, characters 31-32:
# Error: Syntax error
```

Reply:
To avoid the double dune bound, you should delete this line: https://github.com/savonet/ocaml-lastfm/blob/51684e16c743a9924363f41d70470c6ff191a470/dune-project#L17

Reply:
Fixed, thank you!